jwst/lib/tests/test_set_telescope_pointing.py

```diff
@@ -243,10 +243,10 @@ def test_add_wcs_default(data_file):
     assert model.meta.wcsinfo.cunit2 == 'deg'
     assert np.isclose(model.meta.wcsinfo.cdelt1, 3.0555555e-5)
     assert np.isclose(model.meta.wcsinfo.cdelt2, 3.0555555e-5)
-    assert np.isclose(model.meta.wcsinfo.pc1_1, -1.0)
-    assert np.isclose(model.meta.wcsinfo.pc1_2, 0.0)
-    assert np.isclose(model.meta.wcsinfo.pc2_1, 0.0)
-    assert np.isclose(model.meta.wcsinfo.pc2_2, 1.0)
+    assert np.isclose(model.meta.wcsinfo.pc1_1, -0.7431448254773942)
+    assert np.isclose(model.meta.wcsinfo.pc1_2, 0.6691306063588582)
+    assert np.isclose(model.meta.wcsinfo.pc2_1, 0.6691306063588582)
+    assert np.isclose(model.meta.wcsinfo.pc2_2, 0.7431448254773942)
```

Comment:
These changes in the PC matrix seem pretty large for what is expected to be a pretty small change in the pointing.  The old pointing was essentially no rotation on the sky, and this new one shows ~42 degrees rotation.  Same for following 3 tests.  Do we understand this?

Reply:
For these unit tests I think the `v3yangle` is 42 deg across the board, so I think that that accounts for it?